pkg/watch/mux.go

```diff
@@ -22,6 +22,15 @@ import (
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/runtime"
 )
 
+// FullChannelBehavior controls how the Broadcaster reacts if a watcher's watch
+// channel is full.
+type FullChannelBehavior int
+
+const (
+	WaitIfChannelFull = iota
+	DropIfChannelFull = iota
```

Comment:
Please type these, also iota means you don't have to say anything on the second line.

```
	WaitIfChannelFull FullChannelBehavior = iota
	DropIfChannelFull
```